projects/admin/src/app/record/detail-view/vendor-detail-view/vendor-detail-view.component.html

```diff
@@ -0,0 +1,69 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019 RERO
+  
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+  
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+  
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record$ | async as record">
+  <h1 class="mb-3">{{ record.metadata.name }}</h1>
+  <article>
+    <section class="m-2 p-2">
+      <!-- WEBSITE -->
+      <dl class="row mb-0" *ngIf="record.metadata.website">
+        <dt class="col-sm-3 offset-sm-2 offset-md-0">
+          {{ 'Website' | translate }}:
+        </dt>
+        <dd class="col-sm-7 col-md-8 mb-0">
+          <a target="_blank" href="{{ record.metadata.website }}">{{ record.metadata.website }}</a>
```

Comment:
The use of target _blank is not usually recommended. It would be good to have a decision at the project level to decide the standard way to make an external link.

Reply:
ok